olive/passes/openvino/reshape.py

```diff
@@ -0,0 +1,146 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Intel Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from pathlib import Path
+from typing import Dict,Type, Union
+import shutil
+from olive.hardware.accelerator import AcceleratorSpec
+from olive.model import OpenVINOModelHandler
+from olive.passes import Pass
+from olive.passes.pass_config import BasePassConfig, PassConfigParam, get_user_script_data_config
+
+
+class OpenVINOReshape(Pass):
+    """Reshapes OpenVino Model."""
+
+    @classmethod
+    def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
+        return {
+            **get_user_script_data_config(),
+            "extra_configs": PassConfigParam(
+                type_=Dict,
+                default_value=None,
+                required=False,
+                description=(
+                    "Extra configurations for OpenVINO model conversion. extra_config can be set by "
+                    "passing a dictionary where key is the parameter name, and the value is the parameter value. "
+                    "Please check Conversion Parameters documentation for more details: "
+                    "https://docs.openvino.ai/2023.3/openvino_docs_OV_Converter_UG_Conversion_Options.html"
```

Comment:
Do we want to use latest `https://docs.openvino.ai/2025/openvino-workflow/model-preparation/conversion-parameters.html` instead?